compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -1605,7 +1611,13 @@ void OMR::Power::MemoryReference::accessStaticItem(TR::Node *node, TR::SymbolRef
          if (!GCRnode)
             GCRnode = cg->getCurrentEvaluationTreeTop()->getNode();
```

Comment:
same comment here

Reply:
Same concern as voiced in previous comment.